packages/mobile/locales/es-419/sendFlow7.json

```diff
@@ -19,7 +19,7 @@
   "sentTo": "Enviado a",
   "withdrawnTo": "Transferido a",
   "receivedFrom": "Recibido de",
-  "accountNumberLabel": "Dirección de cuenta",
+  "accountAddressLabel": "Dirección de cuenta",
```

Comment:
We should only touch the `en-US` json files, otherwise it easily creates conflicts in the Crowdin PR.

Reply:
@jeanregisser should I remove the entire lines: `"accountNumberLabel": "Dirección de cuenta"` and `"accountAddressLabel": "Dirección de cuenta"`?

Reply:
@Jean should be resolved now.

Reply:
No need to delete the key in other locales, we should leave them untouched.